templates/rancher-airgap/overlay/opt/corral/rancher/install-rancher.sh

```diff
@@ -4,20 +4,18 @@ set -ex
 helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
 helm repo update
 
-CORRAL_rancher_host=${CORRAL_rancher_host:="${CORRAL_fqdn}"}
+CORRAL_internal_rancher_host=${CORRAL_internal_rancher_host:="${CORRAL_internal_fqdn}"}
 CORRAL_rancher_version=${CORRAL_rancher_version:=$(helm search repo rancher-latest/rancher -o json | jq -r .[0].version)}
 
 helm upgrade \
 --install \
 --create-namespace \
---set hostname="$CORRAL_rancher_host" \
+--set hostname="$CORRAL_internal_rancher_host" \
```

Comment:
https://github.com/rancherlabs/corral-packages/pull/12#discussion_r1214528827
@slickwarren yes. I have script that will update the ingress to have both.